Bio/Align/AlignInfo.py

```diff
@@ -583,8 +580,8 @@ def _get_letter_freqs(
          - to_ignore - Letters we are specifically supposed to ignore.
          - pseudo_count - Optional argument specifying the Pseudo count (k)
            to add in order to prevent a frequency of 0 for a letter.
-         - e_freq_table - An optional argument specifying the expected
-           frequencies for each letter. This is a SubsMat.FreqTable instance.
+         - e_freq_table - An optional argument specifying a dictionary with
+           the expected frequencies for each letter.
```

Comment:
Maybe retain something like "This can be a FreqTable instance."

Reply:
I don't think FreqTables will survive much longer.